queries.sql

```diff
@@ -7,4 +7,69 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon') 
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+/*update the animals table by setting the species column to unspecified*/
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you add your own name logo created by you at the top of the readme file to make your readme attractive and present you as a brand in the software industry 👍 to catch the recruiters' eyes 😊